packages/dapp/src/hooks/usePageNav.ts

```diff
@@ -0,0 +1,28 @@
+import { useState, useCallback } from 'react';
+
+export enum CurrentPage {
+  home = 'home',
+  connect = 'connect',
+  settings = 'settings'
+}
+
+export type UsePageNavHook = [
+  currentPage: CurrentPage,
+  setCurrentPage: Function
+];
+
+let initialCurrentPage = localStorage.getItem("currentPage") as CurrentPage;
+if (!initialCurrentPage) {
+  initialCurrentPage = CurrentPage.home;
+}
+
+export const usePageNav = (): UsePageNavHook => {
+  const [currentPage, setCurrentPage] = useState(initialCurrentPage);
+
+  const switchCurrentPage = useCallback((newCurrentPage) => {
```

Comment:
the same as in the https://github.com/windingtree/org.id-profile-manager/pull/3/files#r796513902